pkg/container/docker/network.go

```diff
@@ -0,0 +1,16 @@
+package docker
+
+import (
+	"strings"
+
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+// NetworkInspect displays detailed information on one or more networks
+func NetworkInspect(networkNames []string, format string) ([]string, error) {
```

Comment:
IMHO their should be a higher level abstraction that dealing with docker format strings outside this package, but also fine to iterate on this.